src/common/types/chainner-scope.ts

```diff
@@ -29,6 +29,8 @@ struct PyTorchModel {
     scale: int(1..),
     inputChannels: int(1..),
     outputChannels: int(1..),
+    modelType: string, // "ESRGAN" | "RealESRGAN" | "RealSRGANv2" | "Swift-SRGAN"
```

Comment:
I wasn't sure if it was preferable to use just "string" or if defining the specific strings was better

Reply:
I would probably leave it as `string` because then you'll have to change less when adding support for another model type.